src/daemon/creporterdaemon.cpp

```diff
@@ -75,14 +75,15 @@ bool CReporterDaemon::initiateDaemon()
     qCDebug(cr) << "Starting daemon...";
 
     if (!CReporterPrivacySettingsModel::instance()->isValid()) {
-        qCWarning(cr) << "Invalid settings";
+        qCWarning(cr) << "Invalid settings, exitting";
```

Comment:
What settings? Is this a file? Can we state that here? Also below comment says exist if settings are missing so instead of invalid does this actually mean that settings file is missing?